apps/prairielearn/src/pages/instructorCourseAdminSettings/instructorCourseAdminSettings.html.ts

```diff
@@ -186,3 +211,63 @@ function CourseDirectoryMissingAlert({
     `;
   }
 }
+
+function EditActions({
+  coursePathExists,
+  courseInfoExists,
+  hasCoursePermissionView,
+  hasCoursePermissionEdit,
+  exampleCourse,
+  urlPrefix,
+  navPage,
+}: {
+  coursePathExists: boolean;
+  courseInfoExists: boolean;
+  hasCoursePermissionView: boolean;
+  hasCoursePermissionEdit: boolean;
+  exampleCourse: boolean;
+  urlPrefix: string;
+  navPage: string;
+}) {
+  if (!coursePathExists || !courseInfoExists || !hasCoursePermissionView) {
+    return;
+  }
+
+  if (!hasCoursePermissionEdit || exampleCourse) {
+    return html`
+      <p class="mb-0">
+        <a href="${urlPrefix}/${navPage}/file_view/infoCourse.json"> View course configuration </a>
+        in <code>infoCourse.json</code>
+      </p>
+    `;
+  } else {
+    return html`
+      <button
+        id="save-button"
+        type="submit"
+        class="btn btn-primary mb-2"
+        name="__action"
+        value="update_configuration"
+      >
+        Save
+      </button>
+      <button
+        id="cancel-button"
+        type="button"
+        class="btn btn-secondary mb-2"
+        onclick="window.location.reload()"
+      >
+        Cancel
+      </button>
+      <p class="mb-0">
+        <a
+          data-testid="edit-course-configuration-link"
+          href="${urlPrefix}/${navPage}/file_edit/infoCourse.json"
+        >
+          Edit course configuration
+        </a>
+        in <code>infoCourse.json</code>
+      </p>
+    `;
+  }
```

Comment:
Nit, it's nice to reduce indentation (this suggestion will need Prettier's help):

```suggestion
  }
  
  return html`
      <button
        id="save-button"
        type="submit"
        class="btn btn-primary mb-2"
        name="__action"
        value="update_configuration"
      >
        Save
      </button>
      <button
        id="cancel-button"
        type="button"
        class="btn btn-secondary mb-2"
        onclick="window.location.reload()"
      >
        Cancel
      </button>
      <p class="mb-0">
        <a
          data-testid="edit-course-configuration-link"
          href="${urlPrefix}/${navPage}/file_edit/infoCourse.json"
        >
          Edit course configuration
        </a>
        in <code>infoCourse.json</code>
      </p>
  `;
```

Reply:
Updated locally in [1f9d924](https://github.com/PrairieLearn/PrairieLearn/pull/9581/commits/1f9d9242b1ab757397ce2a399f68924f58891b8e).